appengine/py_appengine.bzl

```diff
@@ -162,6 +174,11 @@ py_appengine_binary_base = rule(
             ".yaml",
             ".py",
         ])),
+        "dont_overwrite_appengine_config": attr.bool(
```

Comment:
positive names are usually easier -- so maybe 'overwrite_appengine_config' which defaults to True

Also if this field is first class here, it should be explicitly first class on the public py_appengine_binary method, rather  than a side-effect of being in **kwargs